client/asset/btc/btc.go

```diff
@@ -361,6 +382,10 @@ type ExchangeWallet struct {
 	useSplitTx        bool
 	useLegacyBalance  bool
 	segwit            bool
+	legacyRawFeeLimit bool
+	signNonSegwit     TxInSigner
+	estimateFee       func(RPCClient, uint64) (uint64, error)
```

Comment:
The bch rpc for `estimatefee` is not the same as the legacy bitcoind `estimatefee` rpc. The bitcoin RPC at least took a confirmation target argument. Regardless, since it's not a difference between legacy/non-legacy, I figured a custom function is better.